vite.config.ts.js

```diff
@@ -0,0 +1,25 @@
+// vite.config.ts
```

Comment:
Does this need to be excluded/ignored somewhere? 

Seems odd to have bother the TS and the JS checked in. Even outputting the JS for a config file seems off to me, but I'm too unfamiliar with cute setup to know if it's needed or not?

Reply:
Honestly no idea

Reply:
added to gitignore

Reply:
When I setup Vite, I never saw a generated file like this 🤔